specs/ClearBrowsingData.md

```diff
@@ -0,0 +1,233 @@
+
+  /// Specifies trust tokens data.
+  COREWEBVIEW2_BROWSING_DATA_KIND_TRUST_TOKENS = 0X15,
+
+  /// Specifies conversions data (the completion of meaningful user action on 
+  /// the advertiser's web site by a user who has interacted with an ad from that advertiser).
+  COREWEBVIEW2_BROWSING_DATA_KIND_CONVERSIONS = 0X16,
+
+  /// Specifies site data. This browsing data kind 
+  /// is inclusive of COREWEBVIEW2_BROWSING_DATA_KIND_DOM_STORAGE,
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_COOKIES, COREWEBVIEW2_BROWSING_DATA_KIND_MEDIA_LICENSES,
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_PLUGIN, COREWEBVIEW2_BROWSING_DATA_KIND_SITE_USAGE,
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_DURABLE_PERMISSIONS, COREWEBVIEW2_BROWSING_DATA_KIND_EXTERNAL_PROTOCOLS,
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_ISOLATED_ORIGINS, COREWEBVIEW2_BROWSING_DATA_KIND_TRUST_TOKENS, and
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_CONVERSIONS. If on an android OS, site data also includes web app data. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_SITE = 0x17, 
+
+  /// Specifies content in the HTTP cache including images and other files. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_HTTP_CACHE = 0x18,
+
+  /// Specifies download history data. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_DOWNLOAD_HISTORY = 0x19, 
+
+  /// Specifies general autofill form data. 
+  /// This excludes password forms and includes information like: 
+  /// names, street and email addresses, phone numbers, and arbitrary input. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_GENERAL_AUTOFILL = 0x20, 
+
+  /// Specifies password autofill forms data. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_PASSWORD_AUTOFILL = 0x21,
+
+  /// Specifies browsing history data. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_BROWSING_HISTORY = 0x22,
+
+  /// Specifies bookmarks data.  
+  COREWEBVIEW2_BROWSING_DATA_KIND_BOOKMARKS = 0x23,
+
+  /// Specifies settings data.
+  COREWEBVIEW2_BROWSING_DATA_KIND_SETTINGS = 0x24,
+
+  /// Specifies content settings data.
+  COREWEBVIEW2_BROWSING_DATA_KIND_CONTENT_SETTINGS = 0x25,
+
+  /// Specifies local custom dictionary data. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_LOCAL_CUSTOM_DICTIONARY = 0x26,
+
+  /// Specifies profile data that should be wiped to make it look like a new profile. 
+  /// This does not delete account-scoped data like passwords but will remove access
+  /// to account-scoped data by signing the user out. 
+  /// This browsing data kind if inclusive of COREWEBVIEW2_BROWSING_DATA_KIND_SITE,
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_HTTP_CACHE, COREWEBVIEW2_BROWSING_DATA_KIND_DOWNLOAD_HISTORY,
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_GENERAL_AUTOFILL, COREWEBVIEW2_BROWSING_DATA_KIND_PASSWORD_AUTOFILL,
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_BROWSING_HISTORY, COREWEBVIEW2_BROWSING_DATA_KIND_CONTENT_SETTINGS,
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_BOOKMARKS, COREWEBVIEW2_BROWSING_DATA_KIND_SETTINGS,
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_LOCAL_CUSTOM_DICTIONARY.
+  COREWEBVIEW2_BROWSING_DATA_KIND_PROFILE = 0X27,
```

Comment:
The issue I've been finding is that the flags change as more types are added. I am using some chrome browsing data types and some non-chrome browsing data types.  The chrome browsing data types' flags dynamically change as more types are added to the non-chrome browsing data types.  I am thinking it is better to use a list of the browsing data kinds to avoid multiple ClearBrowsingData calls.